snippets/odoo-python-snippets.cson

```diff
@@ -93,61 +93,62 @@
 	'Compute method':
 		'prefix': 'oo_method_compute'
 		'body': """
-@api.one
-@api.depends('$1', ...)
-def _compute_field(self):
-	self.field = $2
-		"""
+			@api.multi
+			@api.depends("${10:one_field}"${20:, "${30:other_field}"}$40)
+			def _compute_${50:${10:one_field}}(self):
+				for s in self:
+					$60
+			"""
 
 	'Onchange method':
 		'prefix': 'oo_onchange_method'
 		'body': """
-@api.onchange('$1', ...)
-def onchange_field(self):
-	vals = {}
-
-	# Remove warning if necessary
-	vals['warning'] = {
-		'title': _('$2')
-		'message': _('$3')
-	}
-
-	vals['value'] = {
-		'field1': $4,
-		'field2': $5,
-	}
-
-	return vals
-		"""
+			@api.multi
+			@api.onchange("${10:one_field}"${20:, "${30:other_field}"}$40)
+			def _onchange_${50:${10:one_field}}(self):
+				${60:return {
+					"domain": {
+						"${10:one_field}": [("$70", "${80:=}", ${90:False})],$100
+					},
+				}}
+			"""
 
 	'Constrains method':
 		'prefix': 'oo_constrains_method'
 		'body': """
-@api.one
-@api.constrains('$1', ...)
-def _check_field(self):
-  if self.field == $2:
-  	raise ValidationError("$2")
-		"""
+			@api.multi
+			@api.constrains("${10:one_field}"${20:, "${30:other_field}"}$40)
+			def _check_${50:${10:one_field}}(self):
+				${60:if ${70:self.${10:one_field} == $80}:
```

Comment:
ok... but that's not what I am saying. If you use `@api.multi`, you should always use `for s in self` (except in some specific cases, like the onchange, where only one record is included in the recordset)